### DISCUSSION TITLE
Continues and categorical input for multivariate time-series classification

### DISCUSSION THREAD

**Discussion by kormilitzin** [timeseriesAI/tsai#220] [Q&A]
Perhaps I overlooked it in the tutorial section, but I'm wondering is there a simple way of using this fantastic library for time-series classification where input comprises both, continuous and categorical variables. For example, is it possible to 'tsai' for [Rossmann Kaggle data](https://www.kaggle.com/c/rossmann-store-sales/data)?

Here is a screenshot of a data sample:

![Screenshot 2021-09-24 at 23 57 00](https://user-images.githubusercontent.com/20815939/134747578-4fbb9e75-d967-45ae-891f-0c9404e75585.png)

Any advice will be highly appreciated.

**Comment by oguiza** [comment]
Hi @kormilitzin,
Sorry for the late reply.
Take a look at https://timeseriesai.github.io/tsai/models.TabModel.html and https://timeseriesai.github.io/tsai/models.TabTransformer.html. I understand this is what you are looking for. 
Please, let me know if you need anything else.
